api/capi/src/nntrainer.cpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) 2019 Samsung Electronics Co., Ltd. All Rights Reserved.
```

Comment:
plz consider to change 2019 > 2020 (all added files)

TODO:
log util (dlog for tizen)

overall lgtm

Reply:
I will implement log later. Thanks.